src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataAnnotationsModelValidator.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
 {
-    public class DataAnnotationsModelValidator : IModelValidator, IClientModelValidator
+    public class DataAnnotationsModelValidator : IModelValidator
```

Comment:
looked like most references to `DataAnnotationsModelValidator` or `DataAnnotationsModelValidator<T>` now use `DataAnnotationsClientModelValidator` and `DataAnnotationsClientModelValidator<T>`.  so I was asking what uses remained.

now see `DataAnnotationsModelValidatorProvider` still exists and is still enabled.  so my comment isn't actionable.